site/docs/id/hosting/vercel.md

```diff
@@ -80,9 +80,24 @@ if (!token) throw new Error("BOT_TOKEN belum diisi");
 
 const bot = new Bot(token);
 
-export default webhookCallback(bot, "http");
+export default webhookCallback(bot, "std/http");
 ```
 
+::: tip [Vercel Edge Functions](https://vercel.com/docs/functions/edge-functions) memberikan dukungan terbatas untuk grammY
+Kamu masih dapat menggunakan paket inti grammY dan beberapa jumlah plugin, tetapi untuk plugin yang lain mungkin tidak kompatibel karena ada dependensi dari Node.js yang belum di dukung oleh [Edge Runtime](https://edge-runtime.vercel.app)-nya Vercel. 
```

Comment:
```suggestion
::: tip [Vercel Edge Functions](https://vercel.com/docs/functions/edge-functions) memiliki dukungan terbatas terhadap grammY
Meski kamu masih bisa menggunakan paket inti grammY dan sejumlah plugin lainnya, namun, beberapa plugin tertentu mungkin tidak akan kompatibel karena [Edge Runtime](https://edge-runtime.vercel.app)-nya Vercel belum mendukung dependensi Node.js secara keseluruhan.
```

Reply:
```
karena [Edge Runtime](https://edge-runtime.vercel.app)-nya Vercel
```

How about your opinions that I write additional word `-nya`?
Because in english translation it's belong to Vercel word (Vercel's).
I felt that part not suitable.

Reply:
Hmm yeah, does not sound natural to me. Just add `Edge Runtime Vercel` is fine.

Reply:
I agree